### PR TITLE
[Channel] Add OctoBot channel 

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: octobot/factories/exchange_factory.py [hunk 1]
```diff
@@ -31,40 +32,24 @@ def __init__(self, octobot, ignore_config=False):
...
-                elif is_trader_simulator_enabled_in_config(self.octobot.config):
-                    exchange_builder.is_simulated()
-                if backtesting is not None:
-                    exchange_builder.is_backtesting(backtesting)
-                exchange_manager = await exchange_builder.build()
-                await init_exchange_chan_logger(exchange_manager.id)
-                self.exchange_manager_ids.append(exchange_manager.id)
-            else:
-                self.logger.error(f"{exchange_class_string} exchange not found")
+            await get_chan_at_id(OctoBotChannelsName.OCTOBOT_CHANNEL.value, self.octobot.bot_id)\
```
Comment (GuillaumeDSM) [inline octobot/factories/exchange_factory.py:41]: should we add a requirement PR to handle this ?

File: octobot/initializer.py [hunk 2]
```diff
@@ -29,9 +29,13 @@ def __init__(self, octobot):
     async def create(self):
         # initialize tentacle configuration
         self.octobot.tentacles_setup_config = get_tentacles_setup_config()
+
         # initialize tools
         self._init_community()
 
+        # create OctoBot channel
+        await self.octobot.global_consumer.initialize()
```
Comment (GuillaumeDSM) [inline octobot/initializer.py:37]: 👍

File: octobot/octobot.py [hunk 3]
```diff
@@ -67,6 +68,9 @@ def __init__(self, config, ignore_config=False, reset_trading_history=False):
         # octobot_api to request the current instance
         self.octobot_api = OctoBotAPI(self)
 
+        # octobot channel global consumer
+        self.global_consumer = OctoBotChannelGlobalConsumer(self)
```
Comment (GuillaumeDSM) [inline octobot/octobot.py:72]: should we nave this with octobot, something like ```octobot_global_consumer``` ?

File: octobot/factories/exchange_factory.py [hunk 4]
```diff
@@ -31,40 +32,24 @@ def __init__(self, octobot, ignore_config=False):
...
-                    .is_rest_only()
-                if is_trader_enabled_in_config(self.octobot.config):
-                    exchange_builder.is_real()
-                elif is_trader_simulator_enabled_in_config(self.octobot.config):
-                    exchange_builder.is_simulated()
-                if backtesting is not None:
-                    exchange_builder.is_backtesting(backtesting)
-                exchange_manager = await exchange_builder.build()
-                await init_exchange_chan_logger(exchange_manager.id)
-                self.exchange_manager_ids.append(exchange_manager.id)
```
Comment (GuillaumeDSM) [inline octobot/factories/exchange_factory.py:65]: Should we generate the exchange ID here in order to ba able to save it ?

File: octobot/factories/exchange_factory.py (hunk 1 above)
Comment (Herklos) [reply octobot/factories/exchange_factory.py:41]: Yes :+1:

File: octobot/octobot.py (hunk 3 above)
Comment (Herklos) [reply octobot/octobot.py:72]: That was the original name but I renamed it because we are in OctoBot class. Do you prefer `octobot_global_consumer` ?

Comment (GuillaumeDSM) [reply octobot/octobot.py:72]: Hum that's right actually !

File: octobot/factories/exchange_factory.py (hunk 4 above)
Comment (Herklos) [reply octobot/factories/exchange_factory.py:65]: No, the exchange id will be generated in OctoBot-Trading `octobot_channel_consumer` and will notify OctoBot channel of the exchange creation with the exchange_id.

Comment (GuillaumeDSM) [reply octobot/factories/exchange_factory.py:65]: Ok, great !

File: octobot/logger.py [hunk 5]
```diff
@@ -259,12 +269,12 @@ async def balance_profitability_callback(
...
-    cryptocurrency: str,
-    symbol: str,
-    trade: dict,
-    old_trade: bool,
+        exchange: str,
+        exchange_id: str,
+        cryptocurrency: str,
+        symbol: str,
+        trade: dict,
+        old_trade: bool,
```
Comment (GuillaumeDSM) [inline octobot/logger.py:277]: why these double indents ?

Comment (Herklos) [reply octobot/logger.py:277]: It's probably a pycharm auto refactor, I'll revert it :+1:

**Pull request merged**